R/pkg/R/SQLContext.R

```diff
@@ -166,9 +166,9 @@ writeToFileInArrow <- function(fileName, rdf, numPartitions) {
       for (rdf_slice in rdf_slices) {
         batch <- arrow::record_batch(rdf_slice)
         if (is.null(stream_writer)) {
-          stream <- arrow::FileOutputStream(fileName)
+          stream <- arrow::FileOutputStream$create(fileName)
           schema <- batch$schema
-          stream_writer <- arrow::RecordBatchStreamWriter(stream, schema)
+          stream_writer <- arrow::RecordBatchStreamWriter$create(stream, schema)
```

Comment:
ARROW-5505 changes their R API. cc @srowen , @HyukjinKwon , @felixcheung

Reply:
@shaneknapp . It seems that our Jenkins doesn't have R Arrow installation and skip these test.

Reply:
This is not Windows/JDK8 issue. Arrow 0.15 has a breaking change in R interface.

Reply:
> @shaneknapp . It seems that our Jenkins doesn't have R Arrow installation and skip these test.

this is correct.  should i infer that we need to add R Arrow to our worker nodes?

Reply:
Actually, I configured AppVeyor to test Arrow related ones in R for now. So, at least it's being tested in AppVeyor.

Yes, it would be nice if we have Arrow R installed in Jenkins workers to test it out; however, I remember it's a bit tricky to install Arrow R library for now (you cannot just install via CRAN but needs some manual preparation - see https://github.com/apache/arrow/tree/master/r#installation).

I am worried if it messes up the current Jenkins worker's environment. If you see any concern, @shaneknapp, you can just don't install it for now. It's being tested via AppVeyor at least so it's neither a must or urgent.

Reply:
actually, R arrow installed easily and w/o issue (surprising, i know!).  :)

```
sknapp@ubuntu-testing:~$ Rscript -e "packageVersion('arrow')"
[1] ‘0.15.1.1’
```

i will install this on the remaining workers later today/tomorrow.

Reply:
Thank you, @shaneknapp !

Reply:
Thank you!